.travis.yml

```diff
@@ -7,22 +7,22 @@ python:
 services:
   - postgresql
 
+env:
+  - DJANGO_SETTINGS_MODULE="authors.settings"
 # Install dependencies
 install:
   - pip install -r requirements.txt
-  - pip install pytest
-  - pip install coveralls
 before_script:
- - psql -c 'create database ah_unsullied_test' -U postgres
+ - psql -c "create database ah_theunsullied;" -U postgres
 
 # Run tests
 script:
-  - python manage.py makemigrations 
-  - python manage.py migrate 
-  - coverage run --source=authors/ manage.py test 
-  - coverage report
+  - export SECRET_KEY="iamasecrettheunsadtheunsulliedtheundead"
```

Comment:
For security purposes, you should never have the secret-key exposed. Please remove it and have it in the environment variables